pkgs/applications/version-management/pijul/pijul.org.context

```diff
@@ -0,0 +1,92 @@
+
+Context:
```

Comment:
Hm, weird, on my branch I was able to just put the revision in and it supposedly checked out the source at that revision when I built it (just failing on the thing you fixed). I'll have to read into this more; my darcs knowledge is very lacking. Thank you for the information.